docs/preview/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/react-dom": "^18.2.7",
     "@types/react-syntax-highlighter": "^15.5.7",
     "eslint": "^8.38.0",
+    "tailwindcss": "^3.3.0",
```

Comment:
@frandiox should this be updated to tailwindcss v4?

Reply:
Ohh, it was missing the dependency...
I'll update it in another PR where I'm already using TW v4 👍